src/usdb_syncer/gui/settings_dialog.py

```diff
@@ -19,11 +19,13 @@ def __init__(self, parent: QWidget) -> None:
         self._browser = self.comboBox_browser.currentData()
         self.groupBox_reencode_video.setVisible(False)
 
-    def _populate_comboboxes(self) -> None:
+    def _populate_comboboxes(self) -> None:  # pylint: disable=too-complex
```

Comment:
We should address lints instead of disabling them. Otherwise, there's no value in using Pylint at all.
In this case, you should be able to build a tuple of all the enums and comboboxes and iterate over it. (Browser will need to be special-cased due to the icon.)